packaging/bundle-validation/validate.sh

```diff
@@ -148,7 +148,7 @@ test_flink_bundle() {
     export HADOOP_CLASSPATH=$($HADOOP_HOME/bin/hadoop classpath)
     $FLINK_HOME/bin/start-cluster.sh
     $FLINK_HOME/bin/sql-client.sh -j $JARS_DIR/flink.jar -f $WORKDIR/flink/insert.sql
-    sleep 10 # for test stability
+    sleep 20 # for test stability
```

Comment:
It takes longer time for the insert SQL to finish when metadata table is enabled.